### PR TITLE
Support for config from files or environment

### PR DESCRIPTION
**Pull request by dcrowe**
Implements #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +48,35 @@ config = Config(
 vrt = VisualRegressionTracker(config)
 ```
 
+#### Or, as JSON config file `vrt.json`
+```json
+{
+    "apiUrl"="http://localhost:4200",
```
Comment (pashidlos) [inline README.md]: `=` -> `:` to have valid json

File: README.md [hunk 2]
```diff
@@ -47,6 +48,35 @@ config = Config(
...
+```
+
+#### Or, as environment variables
+```sh
+VRT_APIURL="http://localhost:4200" \
+VRT_PROJECT="Default project" \
+VRT_APIKEY="tXZVHX0EA4YQM1MGDD" \
+VRT_CIBUILDID="commit_sha" \
+VRT_BRANCHNAME="develop" \
+VRT_ENABLESOFTASSERT=1 \
```
Comment (pashidlos) [inline README.md]: cannot get this env propperty to override config file with `false` value
I'm getting `enableSoftAssert` in console as string and I guess it's treated as `true`
```
Config(apiUrl='http://localhost:4200', ciBuildId='commit_sha', branchName='develop', project='Default project', apiKey='0TK0P0NQP6MNFQQPTYYBN27JRAA5', enableSoftAssert='False') None
```

File: visual_regression_tracker/config.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+        cfg.apply_environment(environment or os.environ)
+
+        try:
+            cfg.check_complete()
+        except MissingConfigurationError as e:
+            raise MissingConfigurationError(
+                e.field,
+                f'Incomplete configuration, {e.field} was not specified.',
+                'Please provide via the contstructor, a config file "vrt.json", or environment variables.'
```
Comment (pashidlos) [inline visual_regression_tracker/config.py]: `contstructor` -> `constructor`

File: visual_regression_tracker/visualRegressionTracker.py [hunk 4]
```diff
@@ -16,14 +17,16 @@ class VisualRegressionTracker:
...
+    def __init__(self, config: Config = None):
         """
         Creates a new VisualRegressionTracker
 
         :param config: The configuration to use.
         """
-        self.config = config
+        self.config = config or Config.default()
         self.headers = {'apiKey': self.config.apiKey}
+        print(self.config, config)
```
Comment (pashidlos) [inline visual_regression_tracker/visualRegressionTracker.py]: config printed twise in case explicit code config was used

File: README.md (hunk 1 above)
Comment (dcrowe) [reply README.md]: Whoops. fixed

File: README.md (hunk 2 above)
Comment (dcrowe) [reply README.md]: 🤦  fixed

File: visual_regression_tracker/config.py (hunk 3 above)
Comment (dcrowe) [reply visual_regression_tracker/config.py]: fixed

File: visual_regression_tracker/visualRegressionTracker.py (hunk 4 above)
Comment (dcrowe) [reply visual_regression_tracker/visualRegressionTracker.py]: removed

Comment (pashidlos) [reply visual_regression_tracker/visualRegressionTracker.py]: actually I found this info really usefull in order to see real config that was used 
gonna think of adding some info logs like we have for Java sdk

**Review by pashidlos — APPROVED** [review]
works like a charm!

**Pull request merged**